openstates/ky/committees.py

```diff
@@ -41,14 +41,43 @@ def scrape(self, chamber, term):
                 links = links + linkz
 
             for link in links:
-                name = re.sub(r'\s+\((H|S)\)$', '', link.text).strip().title()
-                name = name.replace(".", "")
-                comm = Committee(chamber, name)
-                comm_url = link.attrib['href'].replace(
-                    'home.htm', 'members.htm')
-                self.scrape_members(comm, comm_url)
-                if comm['members']:
-                    self.save_committee(comm)
+                self.scrape_committee(chamber, link)
+                
+
+    def scrape_committee(self, chamber, link, parent_comm=None):
+        home_link = link.attrib["href"]
+        name = re.sub(r'\s+\((H|S)\)$', '', link.text).strip().title()
+        name = name.replace(".", "")
+        if "Subcommittee " in name:
+            subcomm = name.split("Subcommittee")[1]
```

Comment:
Huh! Crazy. Well, we mirror the LIS, and if the LIS says they don't explicitly report to anyone then I think you're right that they're a top-level `committee` :D